### ISSUE TITLE
nuget.config support

### ISSUE THREAD

**Issue by cspwizard** [aliasadidev/vscode-npm-gui#62]
Hi! 

It would be killer feature if this plugin would understand nuget.config file and use it to automatically configure sources and all the rest required settings. Instead of manually configuring....

Thanks!

**Comment by dodu2014** [comment]
Yes, it's great to support nuget.config, and configuring custom sources via settings doesn't seem to work yet.

**Comment by Monarch73** [comment]
Whats also missing is support for dotnet/c# ManagePackageVersionsCentrally-Setting, which manages package versions in a single Directory.Packages.Props file. That would be great too....

**Comment by aliasadidev** [comment]
@cspwizard @cspwizard 

will add it

**Comment by aliasadidev** [comment]
> Whats also missing is support for dotnet/c# ManagePackageVersionsCentrally-Setting, which manages package versions in a single Directory.Packages.Props file. That would be great too....

@Monarch73 
will research about that and inform you about that here

**Comment by kamakaze** [comment]
I agree
